### PR TITLE
removing active respondent check for resend email

### PR DESCRIPTION
**Pull request by andrewmil**
Removing a restriction to allow users to have their verification email sent if their status is set to `ACTIVE`

Removed a test related to this

### CODE REVIEW

File: ras_party/controllers/party_controller.py [hunk 1]
```diff
@@ -623,17 +622,12 @@ def resend_verification_email(party_uuid):
     :param party_uuid: the party uuid
     :return: make_response
     """
-    log.debug('attempting to resend verification_email with party_uuid {}'.format(party_uuid))
+    log.debug('Attempting to resend verification_email with party_uuid {}'.format(party_uuid))
```
Comment (rhiGriff) [inline ras_party/controllers/party_controller.py:625]: Does this need to be updated to conform with the logging standards?

Comment (andrewmil) [reply ras_party/controllers/party_controller.py:625]: It does but all the other logging in party service needs to be updated to meet those standards as well I think. Maybe we should add to the technical debt?

**Review by rhiGriff — APPROVED** [review]
Happy now, I think the changes to logging should be raised as technical debt

**Comment by jonmorphic** [comment]
What logging standards?

**Comment by andrewmil** [comment]
https://digitaleq.atlassian.net/wiki/spaces/SDC/pages/69632229/Logging

I don't think they've been well communicated across the service.
Pretty much it boils down to having everything in key-value pairs so splunk can use it.

Since we're using structlog we could update that log line to be;
`log.debug('Attempting to resend verification_email', party_uuid=party_uuid)`

**Review by nherriot — COMMENTED** [review]
Looks good.

**Pull request merged**